src/organisms/navigation/draggable/index.js

```diff
@@ -15,15 +16,16 @@ const DraggableTabs = ({ children, onDragEnd: dragEnd, onTabClose, onResize, col
   const onTabsResize = useOnTabsResize(ref, tabsRef, onResize, [children])
   const [arrowLeft, arrowRight, onScroll] = useNavigationArrows(ref, tabsRef, children, collapsed)
 
-  const handlers = useCallback(() => {
-    onTabsResize()
-    onScroll()
-  }, [collapsed])
-
   useEffect(() => {
     if (!ref.current) return
 
     const container = ref.current
+
+    const handlers = debounce(300, () => {
+      onTabsResize()
+      onScroll()
+    })
```

Comment:
@novykh shouldn't the handler be put into `ref`? doesn't it create a new debounce on every `useEffect`?